static/app/views/replays/detail/layout/focusTabs.tsx

```diff
@@ -24,9 +30,9 @@ function getReplayTabs(organization: Organization): Record<TabKey, ReactNode> {
     [TabKey.CONSOLE]: t('Console'),
     [TabKey.NETWORK]: t('Network'),
     [TabKey.ERRORS]: t('Errors'),
-    [TabKey.TRACE]: hasTraceTable ? null : t('Trace'),
+    [TabKey.TRACE]: !hasTraceTable || isVideoReplay ? null : t('Trace'),
```

Comment:
`isTraceTable` seemed sus to me... shouldn't we show the label if `hasTraceTable` is true?

might be misunderstanding this boolean though

Reply:
The trace table was something else. What we have now is the trace waterfall.

Reply:
ohhh okay let me revert this